docs/sitemap.txt

```diff
@@ -5,6 +5,7 @@ index.md
 	Manual.md
 		Overview.md
 		Running-Meson.md
+			Builtin-options.md
```

Comment:
This should be at the same level as `Running-Meson.md`, not underneath it. That just makes it harder to find.